### DISCUSSION TITLE
Plantable tree for timber

### DISCUSSION THREAD

**Discussion by Jeetu74** [Keriew/augustus#799] [Ideas]
Can we get ability to plant a tree which generates timber as resource ? Sometimes we clear wide area and remove most of timber producing trees.

**Comment by crudelios** [comment]
We thought about this and decided against it because it could effectively prevent invasions.

However, the game now stores where there were trees previously, so we could use that to allow replanting on original tree tiles.

It's something to think about for the future!

**Comment by Jeetu74** [comment]
Couldn't these tree's be  given same property as wall. Walls gets demolished automatically if they block waypoint.

Replanting original tree tile is also a great idea , it will require an extra overlay which shows where trees were planted before.

**Comment by Hipshot-zz** [comment]
An idea here would be to either make a forester that can replant where there used to be trees (like you said), inside an area he covers, like a well and it can also be so this office needs access to reservoir water, even if there were not water there from the beginning. This also limits where trees can be replanted, not only because you need to cut them down first, but also because you can't get reservoir coverage everywhere.

**Reply by Jeetu74** [reply]
I like this idea too , a new 2x2 structure similar to claypit should be doable. It behaves exactly like Claypit except that it generates Wood instead of Clay.

**Reply by MSTVD** [reply]
unless you mean trees, we already have that in the Timber Yard :P

**Reply by Jeetu74** [reply]
More like Alternate Self Sustaining Timber Yard :D
